### PR TITLE
Added missing name in PanopticNet BatchNormalization layer

### PR DESCRIPTION
**Pull request by willgraf**
## What
* Added a name to the batch norm layer in PanopticNet.
* Updated weights files for Applications.

## Why
* Enables models to be loaded by name to easily remove unused semantic heads.

### CODE REVIEW

**Comment by willgraf** [comment]
Confirmed all application weights are loaded and run with data.
